server.go

```diff
@@ -10,7 +10,8 @@ import (
 func main() {
 	s := server.NewServer("localhost")
 
-	s.Run()
+	config := server.Config{UseSmartAuth: true, UseLoggingMiddleware: true}
```

Comment:
I'd prefer that we default the UseSmartAuth to `false` for the main function -- especially since having it set to true requires a bunch of other infrastructure to be present.  Those who need SmartAuth can override it in their own function -- or perhaps we can add a flag or environment variable to override it.  For this PR, I'd like to see it set to `false` -- and the flag/env options can come in another PR if deemed desirable.  Does that seem reasonable?